extensions/objects/models/objects.py

```diff
@@ -1045,3 +1047,67 @@ class DragAndDropPositiveInt(BaseObject):
     default_value = 1
 
     SCHEMA = PositiveInt.SCHEMA
+
+
+class AlgebraicExpression(BaseObject):
+    """Class for algebraic expressions. Stores a unicode string representing a
+    valid algebraic expression.
+    """
+
+    description = 'A unicode string for an algebraic expression.'
+    default_value = 'a'
+
+    SCHEMA = {
+        'type': 'unicode',
+        'validators': [{
+            'id': 'is_valid_math_expression',
+            'algebraic': True
+        }]
+    }
+
+
+class Placeholder(BaseObject):
```

Comment:
Aren't these the 'Identifiers', if so maybe rename these to 'AlgebraicIdentifiers' or something similar?

Reply:
Yes, that does make things simpler! Done.